TodoAppWithSpringboot/src/main/java/com/webonise/model/Todo.java

```diff
@@ -0,0 +1,16 @@
+package com.webonise.model;
+
+import java.io.Serializable;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Todo implements Serializable{
+	
+	private static final long serialVersionUID = 1L;
+	@Id
+	@Getter @Setter private int id;
```

Comment:
while it is okay to use Lombok.why not generate the getters and setters ourselves?